README.md

```diff
@@ -2079,39 +2078,43 @@ to taxes on equity compensation:
 have in a company.
 
-- To measure the potential value of equity, you have to determine or make educated guesses
-  about several things:
-  - **Stock value**: Determined by the value the company will have in the future, and the number
-    of shares you own.
-  - **Vesting and liquidity**: When you will actually receive the shares and when you’ll be able
-    to sell them.
-  - **Tax**: Both the purchase and the sale of stock can require that you pay taxes—sometimes
-    very large amounts.
-    There are several kinds of taxes that may apply when you have stock:
-    income, capital gains, and AMT.
-- 🔹 Knowing how many shares of stock or stock options you have is meaningless unless you
-  know the number of outstanding shares.
-  What matters is the percentage ownership (typically in basis points) of the company the
-  shares represent, including the [details](#counting-shares) of how the total is counted.
-  Some companies don’t volunteer this information unless you specifically ask for it, but
-  it’s always fair to do so, because without it the offer of shares is almost meaningless.
-- 🔹 It is critical to understand dilution and [risk](#stages-of-a-startup) to ascertain the
-  possible future value of equity for the company and its stage.
-  [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets provides some
-  additional thoughts.
+In summary, you have to determine or make educated guesses about several things:
+
+- **Equity value**: Determined by the value the company may have in the future, and the number
+  of shares you may own?
+  - **Percentage ownership**: As we’ve mentioned, knowing how many shares of stock or stock
+    options you have is meaningless unless you know the number of outstanding shares.
+    What matters is the percentage ownership of the company the shares represent, including
+    the [details](#counting-shares) of how the total is counted.
+  - **Risk**: It is critical to understand [risk](#stages-of-a-startup) in the business and dilution
+    to ascertain the possible future value of equity for the company and its stage.
+    [This article](http://codingvc.com/valuing-employee-options/) from Leo Polovets provides some
+    additional thoughts.
+- **Vesting**: When you will receive the equity, as well as whether you’re able to exercise
+  stock options (and pay the associated costs and taxes).
+  And whether you can do all this before your exercise window expires.
+- **Liquidity**: When you’ll be able to sell the shares, and if that is likely to be for a
+  profit at that time.
+  (We talk about liquidity of private stock next.)
+- **Tax**: The tax implications of your possible grant, exercise, vesting, and sale, in terms
+  of ordinary income tax, employment tax, long-term capital gains, and alternative minimum
+  tax.
+
+That’s a lot, and even so, decisions are uncertain, but it is possible to make much more
```

Comment:
This list, while more complete than what it was, still feels so much to me like a great introduction to what will be covered, part of Equity Comp Basics. If it's going to come up again here we need a more complete introduction than "you have to determine or make educated guesses about several things." Why repeat it instead of just directing readers to these places in the Guide, as we do every other time we think a reminder is needed? There actually is a reason to do so, b/c we're heading back into pretty non-technical material after a lot of awful tax material, so this list can almost function like a "settle in, settle down" moment. Not a bad way to think about it. But then it needs a clearer introduction/transition. I know I've said it before and I won't again, but the fact that the new material of this chapter is bookended by two summaries ("Dangers" as well as this list) is a bit confusing, hard to tell whether we really have heard all this before or if we missed something, and if so, what. Potential to induce some panic, especially after taxes, if we don't explain better why we're repeating this stuff.

Reply:
Might make sense to move the "Summary of dangers" up to follow this summary as well, before moving on to new material.